extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -487,6 +493,12 @@ export class AmpA4A extends AMP.BaseElement {
               creativeParts.creative) {
             this.creativeBody_ = creativeParts.creative;
           }
+          if (creativeParts && creativeParts.size
```

Comment:
can this if block be moved below the next one? if so, you can eliminate checking creativeParts here, as it is also checked in the next if